### PR TITLE
Feat/fix yarn test console errors

### PR DESCRIPTION
**Pull request by abhishek-ws**
### Ticket Link

---

### Related Links

---

### Description

 **This PR has following updates** 
- Fixed all console errors while running `yarn test`
- Please view the _Gifs_ below for more info
 
### Steps to Reproduce / Test

- yarn test

### GIF's

**BEFORE**
Fix_Test_errors

**AFTER**
AFTER_Fix_Test_errors

**UI has no issues with the update**
UI_Fix_Test_errors

### CODE REVIEW

File: app/components/T/index.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ T.propTypes = {
   marginBottom: PropTypes.number,
   values: PropTypes.object,
   text: PropTypes.string,
-  type: PropTypes.oneOfType(Object.keys(fonts.style))
```
Comment (alichherawalla) [inline app/components/T/index.js:34]: type: PropTypes.oneOf(Object.keys(fonts.style))

File: app/components/For/index.js [hunk 2]
```diff
@@ -30,11 +30,9 @@ For.propTypes = {
   of: Proptypes.array,
   type: Proptypes.node,
   parent: Proptypes.object,
-  iteratee: Proptypes.string,
   renderItem: Proptypes.func.isRequired,
-  noParent: Proptypes.bool
-};
-For.defaultProps = {
-  isRow: true
+  noParent: Proptypes.bool,
```
Comment (alichherawalla) [inline app/components/For/index.js:34]: update tests

Comment (abhishek-ws) [reply app/components/For/index.js:34]: Updated Accordingly

File: app/components/T/index.js (hunk 1 above)
Comment (abhishek-ws) [reply app/components/T/index.js:34]: Fixed

**Review by alichherawalla — APPROVED** [review]

**Pull request merged**